src/libs/actions/Task.js

```diff
@@ -595,10 +596,14 @@ function getAssignee(details) {
  * */
 function getShareDestination(reportID, reports, personalDetails) {
     const report = lodashGet(reports, `report_${reportID}`, {});
+    let subtitle = '';
+    if (ReportUtils.isChatReport(report) && ReportUtils.isDM(report) && report.participants && report.participants.length === 1) {
```

Comment:
I think this is a good time to create `ReportUtils.hasSingleParticipant` method
We are reusing the same code (`report.participants.length === 1`) in HeaderView, actions/Task and ReportUtils, we could use the new method instead

Reply:
HeaderView, actions/Task and ReportUtils are the only 3 occurrences (you can just global search `participants.length === 1`)

Reply:
I think both are fine, but putting it close to similar methods is slightly better.
Also make sure to add [JSDocs](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)